pallets/polkadex/src/tests.rs

```diff
@@ -24,7 +24,7 @@ fn setup_balances() {
     // Creates first asset to alice's account
     assert_ok!(pallet_generic_asset::Module::<Test>::create_asset(None,Some(alice),options_alice));
     let options_bob = pallet_generic_asset::AssetOptions::<u128, u64> {
-        initial_issuance: 105 * UNIT/100, // Made changes to accommodate additional limit orders
+        initial_issuance: 105 * UNIT/100,
```

Comment:
It should be 1* UNIT.

Reply:
I added to the balance for testing. We need more balance to test the full cycle of iteration. So, I recommend we increase it to 2 BTC if a whole number is needed.

Reply:
@visiondream3 Yes, you can do that in a new setup_balances() functions specific to your test case. if you modify this setup_balance(). We need to redo the entire test cases as they are dependent on it.

Reply:
right, since I'll be creating a separate function I believe it shouldn't cause issues. I"ll use original values in a separate function...

Reply:
Sure, That will do.